snippets/features-support.mdx

```diff
@@ -382,6 +382,20 @@ We may include notes on migrating to newer versions if a new feature warrants re
             - Avoid complex transforms repeatedly on the `<canvas>` elements that display Rive animations (or `<RiveComponent />` in the React runtimes)
             - We recommend using `@rive-app/webgl` to display mesh on Firefox for best performance
     </Accordion>
+    <Accordion title="Caching a Rive File">
+        | **Runtime** | **Version** |
+        | --- | --- |
+        | Web | ✅ Supported |
+        | React | ✅ Supported |
+        | React Native | Not yet supported |
+        | Flutter | ✅  Supported |
+        | Flutter (rive_native) | ✅ Supported |
+        | Apple | ✅ Supported |
+        | Android | ✅ Supported |
+        | C++ | ✅  Supported |
+        | Unity | Not supported |
+        | Unreal | Not supported |
```

Comment:
@Tod-Rive Is this supported or even relevant in Unreal? Should I put "NA" instead since "Not supported"?

Reply:
Confirmed that this is unsupported right now.